docs/application/dotnet/guides/multimedia/media-controller.md

```diff
@@ -201,6 +202,9 @@ The following table lists all the media control metadata properties the client c
 | `Description`  | Description of the latest content in the media control server |
 | `TrackNumber`  | Track number of the latest content in the media control server |
 | `AlbumArtPath` | Album art of the latest content in the media control server |
+| `Season` | Season information of the latest content in the media control server |
+| `Episode` | Episode information of the latest content in the media control server |
+| `Resolution` | Resolution of the latest content in the media control server |
 
```

Comment:
In line no: 211
"* Dependencies" is used for list. To maintain consistency, please change it to "- Dependencies".
- Dependencies

If this is not in your scope to change, please ignore my comments.

Reply:
Done